assembly-ide/src/assembly/tomcat/bin/setenv.sh

```diff
@@ -27,9 +28,6 @@
 [ -z "${CLASSPATH}" ]  && CLASSPATH="${CATALINA_HOME}/conf/:${JAVA_HOME}/lib/tools.jar"
 
 
-export JAVA_OPTS="$JAVA_OPTS  -Dche.logs.dir=${CHE_LOGS_DIR}"
-
-
-#Class path
-[ -z "${SERVER_PORT}" ]  && SERVER_PORT=8080
-export SERVER_PORT
\ No newline at end of file
+#SERVER_PORT
```

Comment:
I now set this in che.sh when catalina.sh is called.  This is necessary so that the port value can be passed in.    On line 371 and 372 of the che.sh - these values are updated.

  [ -z "${JAVA_OPTS}" ]  && export JAVA_OPTS="-Dport.http=${CHE_PORT}"
  [ -z "${SERVER_PORT}" ]  && export SERVER_PORT=${CHE_PORT}